client/components/button/style.scss

```diff
@@ -159,6 +159,7 @@ button {
 
 .button.is-borderless {
 	border: none;
+	background: none;
```

Comment:
We are stripping it here: https://github.com/Automattic/wp-calypso/blob/master/client/components/button/style.scss#L203

For borderless AND compact. Which is probably redundant with this change.